semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py

```diff
@@ -272,7 +272,7 @@ class AttributeType:
     @staticmethod
     def get_type(t):
         if isinstance(t, numbers.Number):
```

Comment:
Technically this could be double. Either we reject double outright (preferable IMHO), or we need to distinguish int and float here and generate int/double accordingly.

Reply:
I'd reject double right away and if one would ever specify floating point numbers as enum values we could just treat them as strings.

Reply:
Changed it to reject doubles in 8374bb4.